### ISSUE TITLE
Permissions of dist/ and dist/translations/

### ISSUE THREAD

**Issue by Eloitor** [williamtroup/Calendar.js#235]
**Describe the bug**
Permissions of dist/ and dist/translations/ are `drwx------`. This makes the contents of this folders unaccessible for my server.

**To Reproduce**
1. Download the calendar-js-2.10.17.zip
2. Create a folder and unzip it there
3. run ls -l to view the permissions

```
wget https://calendar-js.com/downloads/calendar-js-2.10.17.zip
mkdir test
mv calendar-js-2.10.17.zip test
cd test
unzip calendar-js-2.10.17.zip
ls -l
```

**Expected behavior**
Reading permissions for all users is expected.

**Additional context**
I fix it manually using
```
chmod 755 dist dist/translations
```

**Comment by williamtroup** [comment]
Will be fixed in v2.10.18.

**Comment by williamtroup** [comment]
OK, will get those updated as well.
